demo/idxd-init.sh

```diff
@@ -18,10 +18,9 @@ for i in $(accel-config list | jq '.[].dev' | grep "$DEV" | sed 's/\"//g'); do
 
 done
 
-ndev=$(accel-config list --idle | jq '.[].dev' | grep -c "$DEV")
 nwq=4
 
-for (( i = 0; i < ndev; i++ )); do
+for i in $(accel-config list --idle | jq '.[].dev' | grep "$DEV" | grep -E -o "[0-9]+"); do
```

Comment:
nit: I'd propose to either do it by enhancing jq expression (jq supports regexps) or at least by avoiding 2 greps, e.g. `accel-config list --idle | jq '.[].dev' | sed -ne "s/$DEV\([0-9]\+\)/\1/p"`